test/e2e/slurm_test.go

```diff
@@ -895,14 +901,27 @@ func parseSlurmCreateOutput(output []byte, profileName string) (string, string,
 	)
 	re := regexp.MustCompile(pattern)
 
-	matches := re.FindSubmatch(output)
+	matches := re.FindSubmatch(filteredOutput)
 	if len(matches) < 5 {
-		return "", "", "", "", fmt.Errorf("unexpected output format: %s", output)
+		return "", "", "", "", fmt.Errorf("unexpected output format: %s", filteredOutput)
 	}
 
 	return string(matches[1]), string(matches[2]), string(matches[3]), string(matches[4]), nil
 }
 
+func filterWarningLines(output []byte) []byte {
+	lines := strings.Split(string(output), "\n")
+	var filteredLines []string
+	for _, line := range lines {
+		if !strings.Contains(line, "Warning: ") {
```

Comment:
This is not specific enough. We cannot filter out any Warning log line, some might be very useful.

If we want to fix it here, then it needs to be very specific like "Warning: spec.SessionAffinity is ignored for headless"